src/init/args.rs

```diff
@@ -7,8 +7,25 @@ use crate::init::xray_exporter::XRayExporterArgs;
 use clap::{Args, ValueEnum};
 use std::error::Error;
 use std::net::SocketAddr;
+use std::path::PathBuf;
+use std::time::Duration;
 use tower::BoxError;
 
+#[derive(Debug, Args, Clone)]
+pub struct FileExporterArgs {
+    /// File format for export (e.g., "parquet")
+    #[arg(long, env = "ROTEL_FILE_FORMAT", default_value = "parquet")]
+    pub format: String,
+
+    /// Directory where files will be written
+    #[arg(long, env = "ROTEL_FILE_PATH", default_value = "/tmp/rotel")]
```

Comment:
I would expect this to end with `OUTPUT_DIR` similar to `ROTEL_PARQUET_EXPORTER_OUTPUT_DIR`

Reply:
updated to OUTPUT_DIR